validator/engine/validator_test.js

```diff
@@ -655,6 +655,7 @@ function attrRuleShouldMakeSense(attrSpec, rules) {
     // Attribute Spec names are matched against lowercased attributes,
     // so the rules *must* also be lower case or non-cased.
     const attrSpecNameRegex = new RegExp('^[^A-Z]+$');
+
```

Comment:
THIS
WHY WHY WHY
this has been happening to me non-stop!!!! I see it's happening to you too!!!! gahhh!!!!
also aside from my internal screaming, please revert this